### DISCUSSION TITLE
How to modify device ID?

### DISCUSSION THREAD

**Discussion by remloyal** [rustdesk/rustdesk#8500] [Q&A]
[issues](https://github.com/rustdesk/rustdesk/issues/8499)

I want to change the ID of the device. How can I proceed?

**Comment by remloyal** [comment]
I have installed multiple systems and RustDesk on the same computer, but the IDs generated by them are the same. How can I modify the device IDs?

![image](https://github.com/rustdesk/rustdesk/assets/93071492/34822687-0905-4f1f-b710-c4a5436fa9c5)

**Reply by remloyal** [reply]
@rustdesk May I ask how to operate it?

**Reply by xlionjuan** [reply]
For Windows cloned systems:
https://learn.microsoft.com/en-us/windows-hardware/manufacture/desktop/sysprep--generalize--a-windows-installation?view=windows-11
https://serverfault.com/questions/874720/sysprep-didnt-change-the-sid

**Comment by rustdesk** [comment]
This is an ad, https://rustdesk.com/docs/en/self-host/rustdesk-server-pro/
